src/components/footer/template.njk

```diff
@@ -3,17 +3,17 @@
   <div class="govuk-width-container {{ params.containerClasses if params.containerClasses }}">
     {% if params.navigation %}
       <div class="govuk-footer__navigation">
-        {% for item in params.navigation %}
+        {% for nav in params.navigation %}
           <div class="govuk-footer__section">
-            <h2 class="govuk-footer__heading govuk-heading-m">{{ item.title }}</h2>
-            {% if item.items %}
+            <h2 class="govuk-footer__heading govuk-heading-m">{{ nav.title }}</h2>
+            {% if nav.items %}
               {% set listClasses %}
-                {% if item.columns %}
+                {% if nav.columns %}
```

Comment:
The below line will just need updating to `{{ nav.columns }}` and then the tests will pass again